.github/workflows/release-test.yml

```diff
@@ -5,12 +5,32 @@ on:
     branches:
       - master
     paths-ignore:
+      - '.github/**'
+      - '.git/**'
+      - '.gitattributes'
+      - '.idea/**'
+      - '.vscode/**'
+      - '.gitignore'
+      - '.DS_Store'
       - 'docs/**'
+      - 'release/**'
```

Comment:
we do  not have `release/**` dir, may I ask why we need to ignore it?

Reply:
You're right. I just copy the exclude paths in makefile https://github.com/apache/apisix-dashboard/blob/28bca96908dabe4af48b06da8f2aeedbbafbe37c/Makefile#L113-L123